deployment.rb

```diff
@@ -91,6 +91,7 @@ def load_configuration()
     'dynamo_tables_table'         => "#{rack_env}_tables",
     'dynamo_tables_index'         => "channel_id-table_name-index",
     'dynamo_properties_table'     => "#{rack_env}_properties",
+    'throttle_data_apis'          => rack_env == :production,
```

Comment:
It might not be a bad idea to start running Redis in all environments.  This has been a source of pain re: the NetSim UI tests.